lib/tasks/migrate_phone_configurations.rake

```diff
@@ -0,0 +1,9 @@
+namespace :db do
+  namespace :migrate do
```

Comment:
I'm open to other ideas for naming the task. It's a migration of data from one table to another, but this could also confuse folk looking for rails migrations.

Reply:
Assuming this is a one-time only task, I don't think the name matters a whole lot, but in the past, I've seen namespaces like `adhoc` used for one-off tasks. 

What do you think about creating an issue to assign the running of this task to someone, and include the command to run? 

Would there be a reason to keep this code around after the task is run? If not, another issue to remove the code from this PR after the task is run?